js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/Sidebar.tsx

```diff
@@ -320,81 +322,91 @@ export const AssetGraphExplorerSidebar = React.memo(
           </Tooltip>
         </Box>
         <div>
-          <Container
-            ref={containerRef}
-            onKeyDown={(e) => {
-              let nextIndex = 0;
-              if (e.code === 'ArrowDown' || e.code === 'ArrowUp') {
-                nextIndex = indexOfLastSelectedNodeRef.current + (e.code === 'ArrowDown' ? 1 : -1);
-                indexOfLastSelectedNodeRef.current = nextIndex;
-                e.preventDefault();
-                const nextNode =
-                  renderedNodes[(nextIndex + renderedNodes.length) % renderedNodes.length]!;
-                setSelectedNode(nextNode);
-                selectNode(e, nextNode.id);
-              } else if (e.code === 'ArrowLeft' || e.code === 'ArrowRight') {
-                const open = e.code === 'ArrowRight';
-                const node = renderedNodes[indexOfLastSelectedNode];
-                if (!node || 'path' in node) {
-                  return;
-                }
-                setOpenNodes((nodes) => {
-                  const openNodes = new Set(nodes);
-                  if (open) {
-                    openNodes.add(nodePathKey(node));
-                  } else {
-                    openNodes.delete(nodePathKey(node));
+          {loading ? (
+            <Box flex={{direction: 'column', gap: 9}} padding={12}>
+              <Skeleton $height={21} $width="50%" />
```

Comment:
Kind of an unusual height value. Does this match the line-height of the rendered text?